lib/Dialect/FIRRTL/Transforms/GrandCentralSignalMappings.cpp

```diff
@@ -629,9 +605,19 @@ FailureOr<bool> GrandCentralSignalMappingsPass::emitUpdatedMappings(
   }
 
   // (3) Generate updated remote references
-  // Helpers to emit basic 'module.name' XMR's for the mappings.
-  // Use inner_sym for non-ports
+
+  // For instance path references, determine where to start.
+  // This is either the main module or the DUT if any.
+  auto pathTop = circuit.getMainModule();
```

Comment:
Mega nit: I would probably call this `dut` instead of `pathTop` as this may be confusing to our future selves.